### PR TITLE
fix(tar): upgrade to 6.1.2

### PR DESCRIPTION
**Pull request by janpio**

### CODE REVIEW

File: src/packages/sdk/package.json [hunk 1]
```diff
@@ -76,7 +76,7 @@
     "string-width": "^4.2.0",
     "strip-ansi": "6.0.0",
     "strip-indent": "3.0.0",
-    "tar": "^6.0.1",
+    "tar": "^6.1.12",
```
Comment (madebysid) [inline src/packages/sdk/package.json:79]: 6.1.2 is what you want I think: https://www.npmjs.com/package/tar

File: src/pnpm-lock.yaml [hunk 2]
```diff
@@ -676,7 +676,7 @@ importers:
       string-width: ^4.2.0
       strip-ansi: 6.0.0
       strip-indent: 3.0.0
-      tar: ^6.0.1
+      tar: ^6.1.12
```
Comment (madebysid) [inline src/pnpm-lock.yaml:679]: I'm not entirely sure why, but we don't commit pnpm-lock updates, I think the release pipeline does that automatically. (Don't quote me on this)

File: src/packages/sdk/package.json (hunk 1 above)
Comment (janpio) [reply src/packages/sdk/package.json:79]:
```suggestion
    "tar": "^6.1.2",
```

Comment (janpio) [reply src/packages/sdk/package.json]: lol good catch

**Review by sdnts — APPROVED** [review]

**Pull request merged**